extension/src/plots/model/custom.ts

```diff
@@ -52,21 +60,15 @@ export const getCustomPlotPathsFromColumns = (
   return { metrics, params }
 }
 
-const getSpecDataType = (type: string) =>
+export const getDataType = (type: string) =>
   type === 'number' ? 'quantitative' : 'nominal'
 
-export const createSpec = (
```

Comment:
Since this function no longer takes arguments, should we turn it into an object instead?